package.json

```diff
@@ -240,73 +240,74 @@
+		"lint:lockfile": "node ./bin/validate-package-lock.js",
+		"lint:md:docs": "wp-scripts lint-md-docs",
+		"prelint:php": "wp-env run composer \"install --no-interaction\"",
+		"lint:php": "wp-env run composer run-script lint",
+		"lint:pkg-json": "wp-scripts lint-pkg-json . 'packages/*/package.json'",
+		"other:analyze-bundles": "npm run build -- --webpack-bundle-analyzer",
+		"other:changelog": "node ./bin/plugin/cli.js changelog",
+		"other:check-licenses": "concurrently \"wp-scripts check-licenses --prod --gpl2 --ignore=@react-native-community/cli,@react-native-community/cli-platform-ios\" \"wp-scripts check-licenses --dev\"",
+		"other:check-local-changes": "node ./bin/check-local-changes.js",
+		"other:cherry-pick": "node ./bin/cherry-pick.mjs",
+		"other:distclean": "rimraf node_modules packages/*/node_modules",
+		"other:native": "npm run --prefix packages/react-native-editor",
+		"other:precheck-local-changes": "npm run docs:build",
 		"postinstall": "patch-package && node ./patches/patch-xcode.js",
-		"prepublishOnly": "npm run clean:package-types && npm run build:packages",
+		"prepare": "husky install",
+		"prepublishOnly": "npm-run-all clean:package-types build:packages",
 		"publish:check": "lerna updated",
-		"publish:patch": "lerna publish patch --no-private --dist-tag patch",
 		"publish:latest": "lerna publish --no-private",
-		"test": "npm run lint && npm run test-unit",
-		"test:create-block": "bash ./bin/test-create-block.sh",
-		"test-e2e": "wp-scripts test-e2e --config packages/e2e-tests/jest.config.js",
-		"test-e2e:playwright": "playwright test --config test/e2e/playwright.config.ts",
-		"test-e2e:debug": "wp-scripts --inspect-brk test-e2e --config packages/e2e-tests/jest.config.js --puppeteer-devtools",
-		"test-e2e:watch": "npm run test-e2e -- --watch",
-		"test-performance": "wp-scripts test-e2e --config packages/e2e-tests/jest.performance.config.js",
-		"test-php": "npm run lint-php && npm run test-unit-php",
-		"test-php:watch": "wp-env run composer run-script test:watch",
-		"test-unit": "wp-scripts test-unit-js --config test/unit/jest.config.js",
-		"test-unit:date": "bash ./bin/unit-test-date.sh",
-		"test-unit:debug": "wp-scripts --inspect-brk test-unit-js --runInBand --no-cache --verbose --config test/unit/jest.config.js ",
-		"test-unit:update": "npm run test-unit -- --updateSnapshot",
-		"test-unit:watch": "npm run test-unit -- --watch",
-		"pretest-unit-php": "wp-env start",
-		"test-unit-php": "wp-env run tests-wordpress /var/www/html/wp-content/plugins/gutenberg/vendor/bin/phpunit -c /var/www/html/wp-content/plugins/gutenberg/phpunit.xml.dist --verbose",
-		"pretest-unit-php-multisite": "wp-env start",
-		"test-unit-php-multisite": "wp-env run tests-wordpress /var/www/html/wp-content/plugins/gutenberg/vendor/bin/phpunit -c /var/www/html/wp-content/plugins/gutenberg/phpunit/multisite.xml --verbose",
+		"publish:patch": "lerna publish patch --no-private --dist-tag patch",
+		"start": "npm run dev",
```

Comment:
Nice addition 👍🏻

Reply:
Nice one. This will make it easier to switch between Gutenberg and custom block development.